src/rocks_compaction_scheduler.h

```diff
@@ -63,5 +69,8 @@ namespace mongo {
         // We'll compact the prefix if any operation on the prefix reports more than 50.000
         // deletions it had to skip over (this is about 10ms extra overhead)
         static const int kSkippedDeletionsThreshold = 50000;
+
+        // thread for async execution of range compactions
+        CompactionBackgroundJob* _compactionJob;
```

Comment:
std::unique_ptr requires full specification of CompactionBackgroundJob. We have only forward class declaration.